lib/pages/chat/events/reply_content.dart

```diff
@@ -76,7 +77,19 @@ class ReplyContent extends StatelessWidget {
             height: ReplyContentStyle.fontSizeDisplayContent * 2,
             decoration: ReplyContentStyle.prefixBarDecoration(context),
           ),
-          const SizedBox(width: ReplyContentStyle.prefixAndDisplayNameSpacing),
+          const SizedBox(width: ReplyContentStyle.contentSpacing),
+          if (displayEvent.hasAttachment)
+            ClipRRect(
+              borderRadius: ReplyContentStyle.previewedImageBorderRadius,
+              child: MxcImage(
+                event: displayEvent,
+                isThumbnail: true,
+                width: ReplyContentStyle.fontSizeDisplayContent * 2,
+                height: ReplyContentStyle.fontSizeDisplayContent * 2,
+                fit: BoxFit.cover,
```

Comment:
should add placeholder is BlurHash of event

Reply:
Done!:)